### PR TITLE
Sqs keep alive

### PR DESCRIPTION
**Pull request by AlexeyRaga**

### CODE REVIEW

File: antiope-sqs/src/Antiope/SQS.hs [hunk 1]
```diff
@@ -11,8 +11,10 @@ module Antiope.SQS
   , ackMessage
   , ackMessages
   , queueSource
+  , keepAliveMessage
   , forAllMessages
   , forAllMessages'
```
Comment (dsturnbull) [inline antiope-sqs/src/Antiope/SQS.hs:16]: nit: seems like hiding the double prime base function would aid clarity

**Review by dsturnbull — APPROVED** [review]
👌🏻

File: antiope-sqs/src/Antiope/SQS.hs (hunk 1 above)
Comment (dsturnbull) [reply antiope-sqs/src/Antiope/SQS.hs:16]: comment applied to the wrong line, I meant forAllMessages’’

Comment (AlexeyRaga) [reply antiope-sqs/src/Antiope/SQS.hs:16]: Yeah, I didn't need to expose it... Not for now at least... Had an idea of using it in the future, but not sure if it's going to be private or not..

**Pull request merged**